### ISSUE TITLE
E1743 long press still doesn't work with latest pull

### ISSUE THREAD

**Issue by LewisSpring** [lsismeiro/awesome-ha-blueprints#4]
Hey, thanks for the great work! For my E1743 buttons, the long press still doesn't work with this, what could I be doing wrong?

_Originally posted by @timcreatedit in https://github.com/lsismeiro/awesome-ha-blueprints/issues/1#issuecomment-1820416580_

**Comment by LewisSpring** [comment]
Hi Tim @timcreatedit

I don't have this remote so unfortunately can't test for you.
Maybe @elephantmetropolis can provide some insight if possible, :)

Can you confirm whether the actions from the remote and the blueprint are matching?

Cheers :)

**Comment by LewisSpring** [comment]
Turns out I have the remote, will test ASAP :D

**Comment by LewisSpring** [comment]
Just tested, am seeing same behaviour.

Move actions appear the same but unfortunately I don't know how they work.
button_up_long: [move_with_on_off_0_83]
```event_type: zha_event
data:
  device_ieee: [redacted]
  unique_id: [redacted]
  device_id: [redacted]
  endpoint_id: 1
  cluster_id: 8
  command: move_with_on_off
  args:
    - 0
    - 83
  params:
    move_mode: 0
    rate: 83
origin: LOCAL
time_fired: "2023-12-04T12:58:45.437142+00:00"
```
button_down_long: [move_1_83]
```
event_type: zha_event
data:
  device_ieee: [redacted]
  unique_id: [redacted]
  device_id: [redacted]
  endpoint_id: 1
  cluster_id: 8
  command: move
  args:
    - 1
    - 83
    - 0
    - 0
  params:
    move_mode: 1
    rate: 83
    options_mask: 0
    options_override: 0
origin: LOCAL
time_fired: "2023-12-04T13:04:49.296544+00:00"
```
Will test previous version of blueprint

**Comment by LewisSpring** [comment]
Apologies for delay, have tested previous version.
Does not seem to work either, if someone can confirm if it ever worked?

**Comment by LewisSpring** [comment]
Scratch that, we have been provided new actions that seem to work (see #5 )

This issue will get closed once tested and merged

**Comment by LewisSpring** [comment]
For now, resolved by #5 cc @timcreatedit
